source/educators/olx/directory-structure.rst

```diff
@@ -15,57 +34,54 @@ This topic describes the structure of a generic OLX (open learning XML) course.
 For more information about how a specific OLX course is structured, see
 :ref:`The Structure of a Sample Course`.
 
-For more information about how a course exported from Open edX Studio is structured,
-see :ref:`Example of OLX for a Studio Course`.
-
-
 ****************************************
 OLX and Directory File Structures
 ****************************************
 
 All files and subdirectories that comprise your OLX course are stored within
-a single directory.
-
-OLX provides for some flexibility in the directory and file structure
-you use to build your course.
+a single directory, called the ``course/`` directory.
 
 ************************
 Top-level Directory
 ************************
 
-Starting out, it is easiest to create your courseware structure in a
-single file, the ``course.xml file``.
-
-This file can contain your entire course, but in most cases, it is convenient
-to split out large chunks of content into individual files. This is typically
-done either at the level of large components, such as problems or homework
-assignments.
+The top-level ``course.xml`` file *can* contain your entire course, but in
```

Comment:
Split into section, subsection, & unit. Certain components in own file, some in-line, but these are more advanced types, for example ORA & LTI Consumer (check this when updating the course)